content/hi/docs/concepts/containers/container-lifecycle-hooks.md

```diff
@@ -0,0 +1,133 @@
+---
+reviewers:
+- Babapool
+- divya-mohan0209
```

Comment:
I don't think this reviewer block should apply to the localization, so we can remove it.
```suggestion
```